### PR TITLE
[DC-2049] Nightly delete_stale_test_datasets error "drc_ops is still in use"

### PR DESCRIPTION
**Pull request by hiro-mishima**

### CODE REVIEW

File: data_steward/tools/delete_stale_test_datasets.py [hunk 1]
```diff
@@ -97,8 +100,13 @@ def main(first_n):
...
 
     for stale_dataset in datasets_to_delete:
+
         LOGGER.info(f"Running - bq_client.delete_dataset({stale_dataset})")
-        bq_client.delete_dataset(stale_dataset)
+
+        try:
+            bq_client.delete_dataset(stale_dataset)
+        except exceptions.BadRequest as e:
+            LOGGER.warning(f"Failed to delete {stale_dataset}. Message: {e}")
```
Comment (nishanthpp93) [inline data_steward/tools/delete_stale_test_datasets.py:109]: I think `e` may not print the message. Maybe `e.message`?
```suggestion
            LOGGER.warning(f"Failed to delete {stale_dataset}. Message: {e.message}")
```

**Review by nishanthpp93 — COMMENTED** [review]
One small thing to check, but looks good otherwise.

**Review by nishanthpp93 — APPROVED** [review]
I just tested it out and `e` seems to work the same as `e.message`, so it may be how google has defined their exception's `__str__`. Sorry for the trouble, but thanks for updating it!

**Comment by hiro-mishima** [comment]
> I just tested it out and `e` seems to work the same as `e.message`, so it may be how google has defined their exception's `__str__`. Sorry for the trouble, but thanks for updating it!

I did not know we can use `e.message` in this context. This way the statement is more explicit and easier to understand. Thanks for the advice! :)

**Pull request merged**